app/views/wpcc/your_results/index.html.erb

```diff
@@ -2,7 +2,7 @@
 
 <section class="section section--contributions">
   <h2 class="section__heading contributions__heading">2. <%= t('wpcc.contributions.title') %>
-    <span class="section__heading-summary">You: <%= session[:employee_percent] %>%, Your employer: <%= session[:employer_percent] %>%</span>
+    <span class="section__heading-summary"><%= t('wpcc.results.contribution_changes.you') %>: <%= session[:employee_percent] %>%, <%= t('wpcc.results.contribution_changes.employer') %>: <%= session[:employer_percent] %>%</span>
     <span><%= link_to(t('wpcc.edit'), new_your_contribution_path, class: 'section__heading-edit') %></span>
```

Comment:
same comment as above, every file you touch can you try to leave it in a better place

```
<span>
  <%= link_to(t('wpcc.edit'), new_your_contribution_path, class: 'section__heading-edit') %>
</span>
```